linux/keyman-config/README.md

```diff
@@ -115,3 +115,64 @@ You will need the build dependencies as well as the runtime dependencies above
 `sudo apt install dh-python python3-all debhelper help2man`
 
 Run `make deb`. This will build the Debian package in the `make_deb` directory.
+
+## Internationalization
+
+### Create or update i18n template file
+
+Run
+
+```bash
+make update-template
+```
+
+This will create or update the file `locale/keyman-config.pot`.
+
+### Add translations for a new language
+
+To add translations for a new language run (replacing `de_DE` with the desired locale):
```

Comment:
Is the locale a BCP 47 code with underscores?

Reply:
Good question. It's whatever is used on Linux, but what is that? Looking...

Reply:
It's a code confirming to ISO/IEC 15897. According to [Wikipedia](https://en.wikipedia.org/wiki/Locale_(computer_software)) this "is similar to the BCP 47 definition of language tags, but the locale variant modifier is defined differently, and the character set is included as a part of the identifier." (Although the character set isn't always included).

So, yes, I guess it boils down to BCP 47 code with underscores.